requirements-test.txt

```diff
@@ -1 +1,3 @@
+Django==1.9.12
+django-environ==0.4.1
```

Comment:
There some extra dependencies. I think in this variant it's better to create third intermediate file